frontend/packages/console-shared/src/components/editor/CodeEditor.scss

```diff
@@ -40,10 +40,11 @@
     }
   }
 
-  /* Needed to get the CodeEditorToolbar to place buttons on the right side */
-  .pf-v6-c-code-editor__controls, .ocs-yaml-editor-toolbar {
+  .pf-v6-c-code-editor__controls {
     width: 100%;
-    height: 100% !important;
     align-items: center;
+    // the height of a PatternFly standard button.
+    // makes sure all toolbars are of the same height
+    height: 37px;
```

Comment:
This give me pause as it makes a bunch of assumptions about the contents that will invariably lead to future bugs.  Where are you seeing differences in height?

Reply:
when the height is set to `100%`, the switch in `CodeEditorToolbar` will not take up the intended height. 

`37px` is the height+padding measurement of the Shortcuts button. by setting it we make sure that the toolbar is always the same height regardless of its contents

![image](https://github.com/user-attachments/assets/8e9e073e-b4d8-48e7-81fe-21e9c03553ff)

Reply:
Ah.  So maybe we do a `min-height: calc(var(--pf-v6-c-button--LineHeight, inherit) + var(--pf-v6-c-button--PaddingBlockStart) + var(--pf-v6-c-button--PaddingBlockEnd))` so at least that number calculated?

Reply:
fair point, i've set it to use the global tokens

```css
    min-height: calc(
      1em * var(--pf-t--global--font--line-height--body) +
      2 * var(--pf-t--global--spacer--control--vertical--default)
    );
```